Members/Medhir Manoj/cLog.md

```diff
@@ -1 +1,2 @@
-
+##### [commit1] (https://github.com/Shady2320/Codeforces/commit/d9beee08f036073a90ae7a528b8e99786962d49f)
```

Comment:
there should be no space between ] and (  please change the above to:

##### Day 1:
* [commit1](https://github.com/Shady2320/Codeforces/commit/d9beee08f036073a90ae7a528b8e99786962d49f)